### ISSUE TITLE
All CxJS samples stopped working

### ISSUE THREAD

**Issue by mstijak** [CompuIves/codesandbox-client#1446]
# 🐛 bug report

## Description of the problem
All CxJS samples stopped working with error:

**SyntaxError**
Unexpected identifier on the first line.

## How has this issue affected you? What are you trying to accomplish?
Examples should run properly. [CxJS homepage](https:://cxjs.io) features examples from CodeSandbox.

###  Link to sandbox: https://codesandbox.io/s/42n5l3z3xw

Any ideas what has changed? CxJS still uses Babel 6. Could that be an issue?

**Comment by CompuIves** [comment]
That's really weird, looking into it now!

**Comment by CompuIves** [comment]
Found the issue, I changed some default behaviour related to compiling node_modules back. When enabling it for cxjs the error went away. I'm adding the cxjs sandbox to the test suite so this won't happen again. Fix should be live in a couple minutes!

**Comment by mstijak** [comment]
Thank you.

On Jan 9, 2019 17:07, "Ives van Hoorne" <notifications@github.com> wrote:

Found the issue, I changed some default behaviour related to compiling
node_modules back. When enabling it for cxjs the error went away. I'm
adding the cxjs sandbox to the test suite so this won't happen again. Fix
should be live in a couple minutes!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/CompuIves/codesandbox-client/issues/1446#issuecomment-452751247>,
or mute the thread
